ui/src/lib/header.svelte

```diff
@@ -0,0 +1,34 @@
+<script>
+    import {
+		Navbar,
+		NavBrand,
+		NavHamburger,
+		Avatar,
+		Dropdown,
+		DropdownItem,
+		DropdownHeader,
+		DropdownDivider,
+	} from 'flowbite-svelte';
+</script>
+<Navbar let:hidden let:toggle color="red">
+	<NavBrand href="/">
+		<img src="salt-shaker.png" class="mr-3 h-6 sm:h-9" alt="Flowbite Logo" />
+		<span class="self-center whitespace-nowrap text-xl font-semibold dark:text-white"
+			>Salt Mines</span
```

Comment:
Are we intending to call the site Salt Mines? If not this should probably match up with the name of the site.

Reply:
No clue what this is really being called. I figured Salt Mines could at least be the code name for the project.

Reply:
I don't expect any of these styles to be kept.

Reply:
Sounds good to me! Let's get mining!